src/js/components/__tests__/__snapshots__/components-test.js.snap

```diff
@@ -2305,8 +2305,12 @@ submit",
     "_calculateChangedBits": null,
     "_currentRenderer": null,
     "_currentRenderer2": null,
-    "_currentValue": undefined,
-    "_currentValue2": undefined,
+    "_currentValue": Object {
+      "useFormContext": [Function],
+    },
+    "_currentValue2": Object {
+      "useFormContext": [Function],
+    },
```

Comment:
Is having Objects as values in this snapshot were what you anticipated?

Reply:
Yes indeed.